spring-boot-docs/src/main/asciidoc/appendix-application-properties.adoc

```diff
@@ -846,6 +846,63 @@ content into your application; rather pick only the properties that you need.
     spring.jms.template.receive-timeout= # Timeout to use for receive calls in milliseconds.
     spring.jms.template.time-to-live= # Time-to-live of a message when sending in milliseconds. Enable QoS when set.
 
+	# APACHE KAFKA ({sc-spring-boot-autoconfigure}/kafka/KafkaProperties.{sc-ext}[KafkaProperties])
+	# Note that these properties (hyphenated or camelCase map directly to the Apache Kafka dotted properties, refer
+	# to the Apache Kafka documentation for details).
+	# The first few of these properties apply to both producers and consumers, but can be specified at the producer or
+	# consumer level.
+	# Apache Kafka designates properties with an importance: HIGH, MEDIUM and LOW.
+	# Spring Boot auto configuration supports all HIGH importance properties, some selected MEDIUM and LOW,
+	# and any that do not have a default value.
+	# To set additional properties, override the consumer and/or producer factory beans; see <<kafka-extra-props>>.
```

Comment:
I would rather see that text in the documentation itself. We really intend this appendix to only list the keys and their documentation (we have the intend to actually auto-generate it).